### PR TITLE
Adds info prints, SC3 logs backups, db exports, specific SC3 install …

### PR DESCRIPTION
**Pull request by FMassin**
…and specific data dir. As well as two utilities:
- event.py can be used to make a list of event ID to be played-back afterwards,
- plot.py makes a rudimentary timeline plot of the results, current version being still very specific to Mvs, to be updated soon.

### CODE REVIEW

File: misc/event.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+def event(baseurl='IRIS', 
+		starttime=None, 
+		endtime=None,
+		minmag=3., 
+		maxmag=10., 
+		maxnumber=10, 
+		catalog=None):
+
+	from obspy import UTCDateTime
```
Comment (yannikbehr) [inline misc/event.py]: Is there a particular reason to have the imports inside the function? I don't want to impose pep8 on you but I think their guidelines on imports and indenting make a lot of sense https://www.python.org/dev/peps/pep-0008/#imports

File: misc/event.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		print('Auto time limit:',file=sys.stderr)
+		print(starttime,file=sys.stderr)
+		print(endtime,file=sys.stderr) 
+		#print("time conversion failed")
+                #sys.exit()
+	
+	if catalog:
+		cat = client.get_events(limit=maxnumber, orderby="magnitude",starttime=starttime, endtime=endtime, minmagnitude=minmag, maxmagnitude=maxmag, catalog="ISC")
+	else:
+		cat = client.get_events(limit=maxnumber, orderby="magnitude",starttime=starttime, endtime=endtime, minmagnitude=minmag, maxmagnitude=maxmag)
```
Comment (yannikbehr) [inline misc/event.py]: What's the reason to hardwire the request to the ISC catalog?

File: playback.sh [hunk 3]
```diff
@@ -3,14 +3,15 @@
 PLAYBACKROOT="$( dirname "$( readlink -f "$0")")/"
 MAKEMSEEDPLAYBACK="${PLAYBACKROOT}make-mseed-playback.py"
 RUNPLAYBACK="${PLAYBACKROOT}playback.py"
+PLAYBACKDATA="${PLAYBACKROOT}data"
```
Comment (yannikbehr) [inline playback.sh]: This requires someone to remember the trailing '/'. I think two '/' are not a problem but a missing one is.

File: playback.sh [hunk 4]
```diff
@@ -285,28 +293,57 @@ if [ $PREPARATION != "false" ]; then
...
+
+	# run the playback
+	if [ -z "$EVENTID" ] && [ -z "$FILEIN" ]
+	then	
+		# continuous data 
 		MSFILE=`ls "${PBDIR}"/*sorted-mseed`
         	EVNTFILE=`ls "${PBDIR}"/*_events.xml`
-        	"$RUNPLAYBACK"  "${PBDIR}/${PBDB}" "${MSFILE}" "${DELAYS}" -c "${CONFIGDIR}" -m ${MODE} -e "${EVNTFILE}"
-	
+		"$RUNPLAYBACK"  "${PBDIR}/${PBDB}" "${MSFILE}" "${DELAYS}" -c "${CONFIGDIR}" -m ${MODE} #-e "${EVNTFILE}"
```
Comment (yannikbehr) [inline playback.sh]: What is the rational to take out the eventfile?

**Review by yannikbehr — COMMENTED** [review]
Hi Fred, thanks for the pull request. Very nice new features! I just had a few minor comments

File: misc/event.py (hunk 1 above)
Comment (FMassin) [reply misc/event.py]: Totally agree.

File: misc/event.py (hunk 2 above)
Comment (FMassin) [reply misc/event.py]: My mistake.

File: playback.sh (hunk 4 above)
Comment (FMassin) [reply playback.sh]: I was trying something and forgot about it, sorry...

**Comment by FMassin** [comment]
Healthy review Yannik, thanks.

File: playback.sh (hunk 3 above)
Comment (FMassin) [reply playback.sh]: Corrected.

**Pull request merged**